specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5214 @@
+          "description": "The ArmId of the keyVault where the customer owned encryption key is present.",
+          "type": "string"
+        },
+        "keyIdentifier": {
+          "description": "Key vault uri to access the encryption key.",
+          "type": "string"
+        },
+        "identityClientId": {
+          "description": "For future use - The client id of the identity which will be used to access key vault.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "keyIdentifier",
+        "keyVaultArmId"
+      ],
+      "type": "object"
+    },
+    "LinkedWorkspace": {
+      "description": "Linked workspace.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ResourceId of the link of the linked workspace."
+        },
+        "name": {
+          "type": "string",
+          "description": "Friendly name of the linked workspace."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of linked workspace."
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceDto": {
+      "type": "object",
+      "description": "object used for creating linked workspace.",
+      "properties": {
+        "name": {
+          "description": "Friendly name of the linked workspace",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceProps": {
+      "type": "object",
+      "description": "LinkedWorkspace specific properties.",
+      "properties": {
+        "linkedWorkspaceResourceId": {
+          "description": "ResourceId of the link target of the linked workspace.",
+          "type": "string"
+        },
+        "userAssignedIdentityResourceId": {
+          "description": "ResourceId of the user assigned identity for the linked workspace.",
+          "type": "string"
+        }
+      }
+    },
+    "ServiceResource": {
+      "type": "object",
+      "description": "Machine Learning service object wrapped into ARM resource envelope.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "description": "Service properties",
+              "$ref": "#/definitions/ServiceResponseBase"
+            }
+          }
+        }
+      ]
+    },
+    "ServiceResponseBase": {
+      "description": "The base service response. The correct inherited response based on computeType will be returned (ex. ACIServiceResponse)",
+      "required": [
+        "computeType"
+      ],
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The service description.",
+          "type": "string"
+        },
+        "kvTags": {
```

Comment:
Is this different than resource tags?  Resource tags are at the top level if they are supported.  Isn't this a proxy resource anyway that wouldn't have tags?

Reply:
these are different than resource tags.